File: submissions/abc165/b.cpp
/**
*    author:  kaneshige
*    created: 31.08.2021 02:56:47
**/

#include <bits/stdc++.h>

using namespace std;

int main() {
    long long n = 100;
    long long b = 100;
    int cnt = 0;
    long long x;
    cin >> x;
    while (n < x)
    {
        n += n/b;
        cnt ++;
    }
    cout << cnt << endl;
}

File: submissions/abc165/b.py
"""
" author : kaneshige
" created : 25.02.2022 21:18:13
"""

import sys
import itertools
from collections import deque


def main():
    x = int(readline())

    cnt = 0
    b = 100
    while b < x:
        b += b//100
        cnt += 1

    print(cnt)

if __name__ == "__main__":
    "----------Constants------------"
    INF = (1 << 62) - 1
    MOD = 10 ** 9 + 7
    # MOD = 998244353
    # sys.setrecursionlimit(10**6)
    true_ceil = lambda x, y: (x + y - 1) // y
    "----------Input------------"
    readline = sys.stdin.readline
    "----------Run------------"
    main()
